### DISCUSSION TITLE
Turning Resea into a MVP (minimum viable product)

### DISCUSSION THREAD

**Discussion by Sologix** [nuta/resea#55] [Ideas]
The first question would be: what target would Resea need to target. Desktop? Server? Embedded?

**Comment by nuta** [comment]
I think the most suitable field is the embedded world. Since Resea is sufficiently matured for my hobby purpose, I have no plans to do next by the way.

**Reply by Sologix** [reply]
Why are you building a graphics framework if you don't plan to turn it into something useful? For your own interest?

**Reply by nuta** [reply]
> For your own interest?

Exactly :)
